client/dynamic-water.lua

```diff
@@ -44,8 +44,8 @@ local threadWait = 100
 RegisterCommand('flood', function(source, args)
     Citizen.CreateThread(function()
         local pCoords, wCoords, allPeds = nil, nil, nil
-        local ped = PlayerPedId()
         while true do
+            ped = PlayerPedId()
```

Comment:
Can you explain how this helps? The value of `ped` shouldn't change within the context of this.

Reply:
Yes of course, com pleasure :).

Basically if one is respawned everything works correctly, the same ped_id will always be used

if one does respown then the ped_id changes them.